tests/test_evaluation.py

```diff
@@ -119,18 +119,18 @@ def test_evaluate_factuality_classifier():
     )
     assert agg_metrics["summaries"]["total"] == 100
     assert agg_metrics["summaries"]["factual"] == 0.48
-    assert agg_metrics["summaries"]["non_factual"] == 0.26
-    assert agg_metrics["summaries"]["non_factual_extrinsic"] == 0.19
+    assert agg_metrics["summaries"]["non_factual"] == 0.27
+    assert agg_metrics["summaries"]["non_factual_extrinsic"] == 0.2
     assert agg_metrics["summaries"]["non_factual_intrinsic"] == 0.1
-    assert agg_metrics["summaries"]["skipped"] == 0.26
+    assert agg_metrics["summaries"]["skipped"] == 0.25
```

Comment:
i feel like we're flutuating a lot here, and am not sure why -- this is all on bart

Reply:
This was left over from the previous changes we introduced since we didn't rerun the classifier: https://github.com/danton-nlp/factual-beam-search/commits/main/results/fbs-logs/bart-test-extrinsic-classifier-knnv1.json